sway-lsp/src/core/session.rs

```diff
@@ -309,7 +308,8 @@ pub fn compile(
         true,
         engines,
         retrigger_compilation,
-        experimental,
+        &[],
```

Comment:
Does this mean LSP will not be supporting experimental features, or I am understanding the code wrongly? What about the experimental features that change the parsing, add new language elements etc like `&self`?

Reply:
> Does this mean LSP will not be supporting experimental features